{
  "file_data": [
    {
      "desc": {
        "cause": "Under Investigation",
        "cluster": false,
        "cust_a": {
          "masked": true,
          "val": 4
        },
        "etr": "2020-11-28T18:30:00-0400",
        "start": "2020-11-28T12:19:43-0400"
      },
      "geom": {
        "p": [
          "elmoG~n_cK"
        ]
      },
      "id": "2",
      "title": "Outage Information"
    }
  ],
  "file_title": "030322"
}
